myuw/static/vue/components/_common/course/meeting-info.vue

```diff
@@ -3,27 +3,27 @@
     <table class="w-100" style="table-layout: fixed;">
       <thead class="sr-only">
         <tr>
-          <th v-if="hasEosDates"
-              :id="'dates-' + sectionId"
+          <th v-if="section.hasEosDates"
+              :id="'dates-' + section.id"
```

Comment:
when using variables in string ``"`days-${section.id}`"`` this format is preferred in JS